src/crawlee/memory_storage_client/_request_queue_client.py

```diff
@@ -15,7 +14,6 @@
 from crawlee._types import StorageTypes
 from crawlee._utils.crypto import crypto_random_object_id
 from crawlee._utils.data_processing import (
-    filter_out_none_values_recursively,
```

Comment:
Maybe we can get rid of this helper completely?

Reply:
Deleted it. It really wasn't used anywhere else